src/app/search-bar/search-bar.component.ts

```diff
@@ -75,6 +75,20 @@ export class SearchBarComponent implements OnInit, AfterViewInit {
     }
   }
 
+  onClick() {
+    if (this.searchdata.fq !== '') {
+      this.store.dispatch(new queryactions.QueryServerAction({'query': this.searchdata.query, start: 0, rows: this.searchdata.rows, fq: this.searchdata.fq}));
+    } else {
+      this.store.dispatch(new queryactions.QueryServerAction({
+        'query': this.searchdata.query,
+        start: 0,
+        rows: this.searchdata.rows
+      }));
+    }
```

Comment:
Have you indented this code? Just asking, to make sure there are no new any changes inside `onClick()`